gst/tensor_filter/tensor_filter.c

```diff
@@ -1065,9 +1071,8 @@ gst_tensor_filter_fixate_caps (GstBaseTransform * trans,
   /**
    * To get the out-caps, GstTensorFilter has to parse tensor info from NN model.
    */
-  gst_caps_unref (othercaps);
 
-  result = gst_tensor_filter_transform_caps (trans, direction, caps, NULL);
+  result = gst_tensor_filter_transform_caps (trans, direction, caps, othercaps);
   result = gst_caps_make_writable (result);
```

Comment:
I wonder no need to uref othercaps? Cause it is unreferenced previously, but after this pr, it is not unreferenced.

Reply:
At ```gst_tensor_filter_transform_caps ()::1047``` of this file, it unrefs othercap if it's not NULL and valid. 

Anyway, we may need to unref it even if the size is 0 if not othercap is non-NULL.

Reply:
I'll fix it by removing unref from gst_tensor_filter_transform_caps and adding unref to gst_tensor_filter_fixate_caps

Reply:
Ah.. yes you're right. I'll add unref